mini-project/BillManagementSystem/accountings/models.py

```diff
@@ -0,0 +1,40 @@
+from djmoney.models.fields import MoneyField
+from phonenumber_field.modelfields import PhoneNumberField
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class Payment(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
With django, we just do like the following:
```py
user = models.ForeignKey(User, on_delete=models.CASCADE)
```

In this case, `user_id` is generally used for internal representation of the fk id itself, as `user` will be used as object representation.

Refer the following on the return objects when called:
```py
pymt_obj = Payment.objects.all()[0]

pymt_obj.user_id   # This returns a str or int
pymt_obj.user      # This returns an instance of the object
```

Same with other instances of this line throughout this model file.